daemon/main.cpp

```diff
@@ -2346,7 +2346,7 @@ int main(int argc, char **argv)
             } else if (optname == "cache-limit") {
                 if (optarg && *optarg) {
                     errno = 0;
-                    int mb = atoi(optarg);
+                    size_t mb = atoi(optarg);
```

Comment:
i don't think this is the right thing to do. it doesn't match the return value of atoi(), and it obfuscates the actual problem.

instead, you should add L to the 1024's.

Reply:
Done.